sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -193,6 +210,16 @@ def _test_subgroup(K, T, S):
     S = FpSubgroup(f, H)
     _test_subgroup(K, T, S)
 
+    f = FpGroup(F, [x**2, y**3, (x*y)**4])
+    G = [x, y]
+    H = [x*y**2*x*y, y**2*x*y*x, y**-1]
+    K, T = subgroup_quotient(G, H, parent_group=f, homomorphism=True)
+    assert T.domain == K
+    assert T(K.generators) == list(f.generators)
+    G = f.subgroup(G)
+    H = f.subgroup(H)
+    assert K.order() == G.order()/H.order()
```

Comment:
Since this is `test_order`, I think it would more appropriate to just check the order here, with no need for `parent_group` and `homomorphism` - that sort of thing should be tested in `test_subgroup_quotient`

Reply:
We still have no test where `G` is a proper subgroup of `parent_group`. Maybe the test above could be changed.